bisdn/installer/install.sh

```diff
@@ -373,6 +373,34 @@ onie-support $bisdn_linux_mnt
 # point bootloader to kernel image (for u-boot: also copy the kernel to /boot)
 platform_install_bootloader_entry $boot_dev $bisdn_linux_part $bisdn_linux_mnt $fs_type
 
+# <arch>-<vendor>-<platform>-<revision>
+onl_platform="$(onie-sysinfo -p | tr '_' '-')"
+# <arch>-<vendor>-<platform>
+onl_baseplatform="${onl_platform%-r*}"
```

Comment:
It wouldn't hurt adding a comment like "remove everything after '-r'. Not everyone is familiar with this syntax, and it's not easy to search for.

Reply:
Doesn't my comment imply that it isn't?

It is the kind of thing that is obvious if you have sufficient context already. If you don't know that particular shell syntax and you don't know what the revision substring looks like, it is much less so. But I won't block the PR over this.